syntax = "proto3";

message ModifyPlayListRequest {
    int32 user_id = 1;
    int32 song_id = 2;
}

message PlayListResponse {
    int32 response = 1;
}

message GetPlayListRequest {
    int32 user_id = 1;
}

message GetPlayListResponse {
    int32 response = 1;
    repeated int32 songs = 2;
}

service PlayListService {
    rpc Add (ModifyPlayListRequest) returns (PlayListResponse);
    rpc Remove (ModifyPlayListRequest) returns (PlayListResponse);
    rpc Get (GetPlayListRequest) returns (GetPlayListResponse);
}